docs/addons/addon-migration-guide.md

```diff
@@ -1,75 +1,61 @@
 ---
-title: Addon migration guide for Storybook 7.0
+title: Addon migration guide for Storybook 8.0
 ---
 
-Storybook 7 is our first major release in over two years. While Storybook’s addon API has not changed much in the past couple of years, addons require several changes for compatibility with Storybook 7. This guide will walk you through the upgrade process.
+We deeply appreciate the dedication and effort addon creators put into keeping the Storybook ecosystem vibrant and up-to-date. As Storybook evolves to version 8.0, bringing new features and improvements, this guide is here to assist you in migrating your addons from 7.x to 8.0. If you need to migrate your addon from an earlier version of Storybook, please first refer to the [Addon migration guide for Storybook 7.0](https://storybook.js.org/docs/addons/addon-migration-guide).
 
-<Callout variant="info">
+## Updating Dependencies
 
-As we gather feedback from the community, we’ll update this page. We also have a general [Storybook migration guide](../migration-guide.md) if you’re looking for that.
-
-</Callout>
-
-## Dependencies
-
-The first thing to do is upgrade any Storybook dependencies in your project. We release the next version of all our packages on the `next` npm tag, so the easiest thing is to reference that in your `package.json`:
+Begin by updating your Storybook dependencies. Use the `next` tag for pre-release versions, `latest` for the most recent stable release, or specify the version directly.
 
 ```json
 {
   "dependencies": {
-    "@storybook/client-logger": "next"
+    "@storybook/client-logger": "next" // or "latest", or "^8.0.0"
   }
 }
 ```
 
-If you'd rather depend on the latest version of Storybook, you can use the `latest` tag:
+## Key Changes for Addons
```

Comment:
I'd call out dropping node 16 to make sure authors bump up to 18

Reply:
good call